### DISCUSSION TITLE
this line codes look like invalid ?

### DISCUSSION THREAD

**Discussion by tkgkn** [ice-lab/icestark#632] [General]
https://github.com/ice-lab/icestark/blob/e5f6ab011e8ac1dfb36d02a295a5d40b6cec759c/packages/icestark/src/start.ts#L51

In `AppRouter.tsx`, `start` function has an object params, inlcude a key called `reroute`, and the value is `this.handleRouteChange`。

In `start.ts`, there is some codes:
```ts
  // use start.ts#L51
  globalConfiguration.reroute = reroute;
  // but rewrite by start function params above
  Object.keys(options || {}).forEach((configKey) => {
    globalConfiguration[configKey] = options[configKey];
  });
```

**Comment by tkgkn** [comment]
@maoxiaoke

**Comment by GabrielKrupaak** [comment]
its fine
